src/data/postgres/errors.ts

```diff
@@ -0,0 +1,310 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2022
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+export const enum PostgresErr {
+  // Class 00 — Successful Completion
+  _00_SUCCESSFUL_COMPLETION = '00000',
+  // Class 01 — Warning
+  _01_WARNING = '01000',
+  _01_DYNAMIC_RESULT_SETS_RETURNED = '0100C',
+  _01_IMPLICIT_ZERO_BIT_PADDING = '01008',
+  _01_NULL_VALUE_ELIMINATED_IN_SET_FUNCTION = '01003',
+  _01_PRIVILEGE_NOT_GRANTED = '01007',
+  _01_PRIVILEGE_NOT_REVOKED = '01006',
+  _01_STRING_DATA_RIGHT_TRUNCATION = '01004',
+  _01_DEPRECATED_FEATURE = '01P01',
+  // Class 02 — No Data (this is also a warning class per the SQL standard)
```

Comment:
I like the idea of using the PostgreSQL Error Codes ( https://www.postgresql.org/docs/current/errcodes-appendix.html ), but typical any variable starting with underscore means private variable. What do you think ?

Reply:
good comment, I forgot about that. and I think you might have noticed that several error 'names' are repeated, so need some additional naming. I added 'C' for class instead of the underscore. What do you think for that?